src/saltext/vmware/modules/vmc_vm_disks.py

```diff
@@ -0,0 +1,118 @@
+"""
+Salt execution module for VMC vm disks
+Provides methods to Create, Read, Update and Delete virtual disks from the VM.
+"""
+import logging
+import os
+
+from saltext.vmware.utils import vmc_constants
+from saltext.vmware.utils import vmc_request
+from saltext.vmware.utils import vmc_templates
+from saltext.vmware.utils import vmc_vcenter_request
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = "vmc_vm_disks"
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def get(hostname, username, password, vm_id, verify_ssl=True, cert=None):
```

Comment:
**suggestion** this gets the list of disks, correct? If so, this should be `list_`, with `__func_alias__ = {"list_": "list"}`

Reply:
This function returns the list of IDs of disks attached to the given vm
the backend API of this function returns the response as:
![image](https://user-images.githubusercontent.com/83271164/154740712-0f731416-5bf7-4866-9573-bfa3171316ec.png)

Reply:
This is still an issue

Reply:
updated